inc/Calculator.h

```diff
@@ -11,6 +11,8 @@
 #define CALCULATOR_H
 
 #include <iostream>
-// Add your function declarations here.
+// Add your function declarations here test.
```

Comment:
1- Use doxygen style comments as shown on Dune coding standards. Also, use full name so runCalculator instead of runCalc
2- Wrap inside of namespaces